rpc/nodeagent/network/virtualnetwork/moc_nodeagent_virtualnetwork.proto

```diff
@@ -40,7 +29,7 @@ message VirtualNetwork {
 	string id = 2;
 	repeated Ipam ipams = 3;
 	Dns dns = 4 [(sensitive) = true];
-	VirtualNetworkType type = 5;
```

Comment:
What are the consequences of changing this? Do we need to make change in nodeagent? We can avoid that

Reply:
I don't think it matters, there is no need to change in nodeagent, just adding a new field

Reply:
do we want to ignore it?

Reply:
the checks passed, can I get an approval to merge it?

Reply:
@chengzy1994 We would be updating this var in wssdagent right? So changing this would require changing wssdagent code?

Reply:
I don't think it requires code change in wssdagent, don't see any code related to this order. @msftpgayam could you help to confirm it? correct me if I miss anything.

Reply:
I expect code changes in the node agent if we make this change. This field will not be available to the node agent.

What is the need for changing this field?